blueprints/app/files/.eslintrc.js

```diff
@@ -30,7 +30,8 @@ module.exports = {
       excludedFiles: [
         'app/**',
         'addon/**',
-        'tests/dummy/app/**'
+        'tests/dummy/app/**',
+        'addon-test-support/**'
```

Comment:
This should be added in the same way as the other addon specific overrides (since it doesn't apply to apps at all)

Reply:
@rwjblue sorry, not sure if I understood you. This is now inside of if statement `blueprint !== 'app'` which means it will be not included to apps at all. Please correct me if I am wrong.

Reply:
DOH! You are totally correct, sorry for not seeing that.